org.adempiere.base/src/org/compiere/model/MOrderLine.java

```diff
@@ -1063,4 +1064,22 @@ public void clearParent()
 	{
 		this.m_parent = null;
 	}
+
+	/**
+	 * Get the description stripping the Close tag that was created when closing the order
+	 * @return
+	 */
+	public String getDescriptionStrippingCloseTag() {
+		// Strip Close() tags from description
+		String desc = getDescription();
+		if (desc == null)
+			desc = "";
```

Comment:
can return "" here to save a few ms.

Reply:
Ready @hengsin - refactored this method for optimization.